.pre-commit-config.yaml

```diff
@@ -1,11 +1,18 @@
--   repo: git@github.com:keybase/pre-commit.git
-    sha: 'e56c58cc34b81fc814f10c5062c31c9db93376df'
+-   repo: local
```

Comment:
we can actually have local stuff instead of a sep. repo so pulling this all back in to simplify things
flow checking was broken as it doesn't expect parameters so it'd fail sometimes (weirdly)
instead we just shell out and call it